### ISSUE TITLE
Alternatives to this bot?

### ISSUE THREAD

**Issue by avanc** [mlopezr/node-red-contrib-matrixbot#5]
It seems that this was only a proof of concept.
Are there any alternatives to send messages to matrix or any other open source messaging system?

**Comment by skylord123** [comment]
I'm currently using this package and have been considering forking it to update it. I'm just busy with other projects atm.

**Comment by KopfKrieg** [comment]
> I'm currently using this package and have been considering forking it to update it. I'm just busy with other projects atm.

Yeah, same here. File support and E2EE would be great, but I don't even know where to start. Besides that the module still works.

**Comment by skylord123** [comment]
I created my own Matrix node-red plugin and decided to come here to let you know about it. I was waiting for this one to get updated but it seems the maintainer has abandoned it.

https://flows.nodered.org/node/node-red-contrib-matrix-chat

This supports files but I have yet to add e2ee support. Currently I am trying to add all the things I need for my bot to administrate my company's chat server and we don't use e2ee yet.
